commands/serve/web/eRender.js

```diff
@@ -36,6 +36,7 @@ async function renderWithEngine() {
   if (!info.enigma.appId) {
     location.href = location.origin; //eslint-disable-line
   }
+  const params = getParams();
```

Comment:
converting old way of getting `params` which was a simple variable assigned into an IFFE had a bit of impact in `eRender.js` module. 

How can i test this module to check if i haven't broke anything?

Reply:
This one is safe and working correctly. 
Steps that i've done: 
1. link cli + cli-serve from la-vie project to my local changes
2. run bar chart independently
3. check the affected section of the code

below you can see the result which works fine ✅ 


https://user-images.githubusercontent.com/29652890/192508742-3c6535d3-bf5e-4b8d-a6d1-55a724943884.mov